src/pages/ClerkScreen.tsx

```diff
@@ -61,10 +64,7 @@ const ClerkScreen = ({
       />
     )
   }
-  const isVxMark = appMode === 'mark'
-  // const isVxPrint = appMode === 'print'
-  // const isVxMarkAndPrint = appMode === 'mark+print'
-  const isTestDecksAvailable = isLiveMode || (!isLiveMode && isVxMark)
+  const isTestDecksAvailable = isLiveMode || (!isLiveMode && appMode.isVxMark)
```

Comment:
One thing I found that I incorrectly updated in a few places was changing `appMode === 'mark'` to `appMode.isVxMark` like was done here. I noticed that you were using `appMode === 'mark'` to mean "does not support printing", which means it should be `!appMode.isVxPrint`. Which should be used here?

Reply:
Ok. I reviewed that swap, which is 👍 and I'll re-review the rest to ensure they're correct.

Reply:
Reviewed and amended that one change.